spring-integration-core/src/test/java/org/springframework/integration/aop/MethodAnnotationPublisherMetadataSourceTests.java

```diff
@@ -83,6 +87,13 @@ public void payloadAndHeaders() {
 		assertEquals("#args[2]", headerMap.get("bar").getExpressionString());
 	}
 
+	@Test
+	public void expressionsAreConcurrentHashMap() {
+		assertThat("Expressions should be concurrent to allow startup", ReflectionTestUtils.getField(source, "channels"), instanceOf(ConcurrentHashMap.class));
+		assertThat("Expressions should be concurrent to allow startup", ReflectionTestUtils.getField(source, "payloadExpressions"), instanceOf(ConcurrentHashMap.class));
+		assertThat("Expressions should be concurrent to allow startup", ReflectionTestUtils.getField(source, "headersExpressions"), instanceOf(ConcurrentHashMap.class));
```

Comment:
One more code style concern and we will go for merge.
We prefer the code line length not more than `120` symbols.

I'm sharing all these rules with you to give your a chance to contribute more, but in much faster manner.
You won't see similar comments from me nay more afterwards! 😄

Reply:
I appreciate it. Updated.